src/util/dom.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {HTMLElement | Array | String} target
+ * @param {Document} document
+ */
+export const domArray = (target, document) => {
+  if (typeof target === 'string') return Array.from(document.querySelectorAll(target))
+  if (target instanceof HTMLElement) return [target]
+  if (target.jquery) return target.toArray()
```

Comment:
You want to keep _some_ jQuery support around?

Reply:
At the moment it's not quite obvious which methods are private and which ones public.
As we don't want to introduce any issues during the migration I'd keep the compatibility with jQuery.
Later we could do a breaking change and just  throw an error here to get a proper stack trace in case somebody passes an unsupported parameter.

Reply:
Makes sense 👍🏼